### PR TITLE
Fixes bugs that would not allow users to comment or see their reviews

### PR DESCRIPTION
**Pull request by TylerHernandez**

### CODE REVIEW

File: src/main/java/com/google/sps/data/Store.java [hunk 1]
```diff
@@ -1,12 +1,18 @@
-public class Store {
+package com.google.sps.data;
+
+public class Store { // todo: update average rating every time a store gets a new comment
```
Comment (mutisog) [inline src/main/java/com/google/sps/data/Store.java]: Should we create an issue for this TODO?

File: src/main/webapp/review.js [hunk 2]
```diff
@@ -116,9 +116,8 @@ function storeComment(email) {
   params.append('rating', rating.options[rating.selectedIndex].value)
   params.append('content', content);
   params.append('store', localStorage.getItem("store"));
-  console.log(store);
-  console.log("store hit");
   params.append('email', email);
+  window.location.reload();
```
Comment (mutisog) [inline src/main/webapp/review.js]: Add a comment on why this is necessary.

File: src/main/webapp/user.html [hunk 3]
```diff
@@ -21,15 +21,10 @@
...
-
-      <div id="email"></div>
-      <div id="form"></div>
-
-      <div id="login-container"></div>
       
       <div id="Setup">Here is a list of reviews you have made! </div>
       <div class= "inline" id="email"></div>
-      <ul id="list"></ul>
+      <div id="list"></div>
```
Comment (mutisog) [inline src/main/webapp/user.html:27]: Does this div need to contain something?

**Review by mutisog — COMMENTED** [review]
A few comments.

File: src/main/java/com/google/sps/data/Store.java (hunk 1 above)
Comment (yjzhang111) [reply src/main/java/com/google/sps/data/Store.java]: I'm pretty sure the function is already there. It just needs to be pulled from main branch.

File: src/main/webapp/review.js (hunk 2 above)
Comment (yjzhang111) [inline src/main/webapp/review.js]: Since we're reloading the page, line 125-132 (originally line 124-131) would never be used and we can just delete it.

**Review by yjzhang111 — COMMENTED** [review]
Add a few comments. You don't necessarily have to fix them because the changes are already in Search Servlet anyways. It's mostly just getting the newest version of code we have and getting ride of redundant code.

File: src/main/webapp/user.html (hunk 3 above)
Comment (TylerHernandez) [reply src/main/webapp/user.html:27]: no

File: src/main/webapp/review.js (hunk 2 above)
Comment (TylerHernandez) [reply src/main/webapp/review.js]: fixed this location, didn't notice it wasn't at the end of a function

File: src/main/java/com/google/sps/data/Store.java (hunk 1 above)
Comment (TylerHernandez) [reply src/main/java/com/google/sps/data/Store.java]: this file is not being used anywhere, i will be deleting it

File: src/main/webapp/review.js [hunk 4]
```diff
@@ -130,6 +128,7 @@ function storeComment(email) {
   const ratingListElement = document.getElementById('comment-list');
   ratingListElement.appendChild(createListElement(drinkObj));
   clearInput();
+  window.location.reload();//this will reload the page so the review section is shown as updated
```
Comment (mutisog) [inline src/main/webapp/review.js:131]: prefer adding a comment before the relevant line of code, and not at the end of the line, as it's easier to read that way.

**Review by mutisog — APPROVED** [review]
Minor comment, otherwise LGTM

**Review by yjzhang111 — APPROVED** [review]

**Pull request merged**